packages/flutter/lib/src/rendering/sliver_group.dart

```diff
@@ -423,4 +419,8 @@ class RenderSliverMainAxisGroup extends RenderSliver with ContainerRenderObjectM
       child = childAfter(child);
     }
   }
+
+  static double _fixPrecisionError(double number) {
+    return number.abs() < precisionErrorTolerance ? 0.0 : number;
+  }
```

Comment:
Interesting—it seems like we've been catching lots of floating point precision errors recently!

Reply:
I have added a test for this.